modules/distribution-service-aws-s3/src/main/java/org/opencastproject/distribution/aws/s3/AwsS3DistributionServiceImpl.java

```diff
@@ -291,12 +292,8 @@ public void activate(ComponentContext cc) {
       // Create AWS client
 
       s3 = AmazonS3ClientBuilder.standard()
-              .withEndpointConfiguration(new AwsClientBuilder.EndpointConfiguration(endpoint
-                      , regionStr))
-              .withPathStyleAccessEnabled(pathStyle)
-              .withCredentials(provider)
-              .build();
-
+              .withEndpointConfiguration(new AwsClientBuilder.EndpointConfiguration(endpoint, regionStr))
+              .withPathStyleAccessEnabled(pathStyle).withCredentials(provider).build();
```

Comment:
The `withEndpointConfiguration` call is an improvement, but I'm not sure if I agree with combining the last three calls into one line. I don't really think it improves readability.